parser/tests/conftest.py

```diff
@@ -0,0 +1,22 @@
+import time
+
+import pytest
+from neo4j import GraphDatabase
+from neo4j.exceptions import ServiceUnavailable
+
+
+@pytest.fixture
+def neo4j():
+    """waiting for neo4j to be ready"""
+    uri = "bolt://localhost:7687"
+    driver = GraphDatabase.driver(uri)
+    session = driver.session()
+    connected = False
+    while not connected:
+        try:
+            session.run("MATCH () return 1 limit 1")
+        except ServiceUnavailable:
+            time.sleep(1)
+        else:
+            connected = True
+    return driver
```

Comment:
@aadarsh-ram and @BryanH01 you might be interested in this pattern.
When making things run automatically it's one of the tricky part: waiting for third party service to be ready before running.

Here I made it a global fixture because that's a nice way to have it in the integrations tests and use it where needed.